src/marqo/core/unstructured_vespa_index/unstructured_vespa_schema.py

```diff
@@ -54,13 +54,16 @@ def _generate_unstructured_marqo_index(self, schema_name: str) -> UnstructuredMa
             normalize_embeddings=self._index_request.normalize_embeddings,
             text_preprocessing=self._index_request.text_preprocessing,
             image_preprocessing=self._index_request.image_preprocessing,
+            video_preprocessing=self._index_request.video_preprocessing,
+            audio_preprocessing=self._index_request.audio_preprocessing,
             distance_metric=self._index_request.distance_metric,
             vector_numeric_type=self._index_request.vector_numeric_type,
             hnsw_config=self._index_request.hnsw_config,
             marqo_version=self._index_request.marqo_version,
             created_at=self._index_request.created_at,
             updated_at=self._index_request.updated_at,
             treat_urls_and_pointers_as_images=self._index_request.treat_urls_and_pointers_as_images,
+            treat_urls_and_pointers_as_media=self._index_request.treat_urls_and_pointers_as_media,
```

Comment:
Does this supersede `treat_urls_and_pointers_as_images` and deprecate it? Otherwise having both will be a weird interface

Reply:
Yeah it does. I couldn't remove treat_urls_and_pointers_as_images since it would cause backwards compatibility issues, but using only treat_urls_and_pointers_as_media for video and audio link would be weird. Setting either to True would put all urls through the process of identifying whether it is an image, audio, video file.

Reply:
Just a note on this, expected behavior:

1. When both are False: all content is treated as text.
2. When treat_urls_and_pointers_as_images is True and treat_urls_and_pointers_as_media is False: will attempt to process URLs and pointers as images, but not as other types of media (like video or audio).
3. When treat_urls_and_pointers_as_images is False and treat_urls_and_pointers_as_media is True: will attempt to process URLs and pointers as various types of media, including images, videos, and audio.
4. When both are True: This case is effectively the same as when only treat_urls_and_pointers_as_media is True. will attempt to process URLs and pointers as various types of media, including images, videos, and audio (same as the third one).

Reply:
Looks like 3 should be a validation error

Reply:
done